packages/plugin-core/src/components/views/NoteGraphViewFactory.ts

```diff
@@ -173,6 +173,13 @@ export class NoteGraphPanelFactory {
             });
             break;
           }
+
+          case GraphViewMessageEnum.toggleGraphView: {
+            AnalyticsUtils.track(DENDRON_COMMANDS.SHOW_NOTE_GRAPH.key, {
```

Comment:
Instead of conflating with the command telemetry, can we use the same 'GraphViewUsed' ("view
 here, not "panel") convention as with the other review?
 
 type: "GraphTypeChanged"
 state: "Local" / "Full"